tests/accordion-test.ts

```diff
@@ -0,0 +1,605 @@
+                        active: false,
+                    },
+                    { title: "title 3?", body: "body 3", active: true },
+                    { title: "title 4?", body: "body 4", active: true },
+                    { title: "title 5?", body: "body 5", active: false },
+                ]
+            )
+        );
+        let titles = el.querySelectorAll(".acc-title");
+        let bodyElement = el.querySelectorAll(".acc-body");
+        let containerElement = el.querySelectorAll(".container");
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(1);
+
+        containerElement.forEach((_, index) => {
+            if (index === 2) {
+                expect(getComputedStyle(bodyElement[index]).display).to.equal(
+                    "block"
+                );
+            } else {
+                expect(getComputedStyle(bodyElement[index]).display).to.equal(
+                    "none"
+                );
+            }
+        });
+
+        const active = el.querySelectorAll(".acc-active");
+        containerElement[0]?.click();
+        await el.updateComplete;
+        containerElement[1]?.click();
+        await el.updateComplete;
+        containerElement[2]?.click();
+        await el.updateComplete;
+       expect(el.querySelectorAll(".acc-active").length).equal(1);
+       
+       containerElement.forEach((_,index) =>{
+         if(index === 2) {
+            expect(getComputedStyle(bodyElement[index]).display).to.equal("block");
+         }
+         else {
+            expect(getComputedStyle(bodyElement[index]).display).to.equal("none");
+
+         }
+       })
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        el.removeAttribute("nomultiple");
+        await el.updateComplete;
+
+        containerElement[2].click();
+        await el.updateComplete;
+        expect(el.hasAttribute("nomultiple")).to.be.false;
+
+        titles[0].click();
+        await el.updateComplete;
+        titles[1].click();
+        await el.updateComplete;
+        titles[4].click();
+        await el.updateComplete;
+        console.log(el);
+
+        expect(titles[0].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+        expect(titles[1].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+        expect(titles[2].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+        expect(titles[3].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[4].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+
+        active.forEach((body, index) => {
+            if (index === 0 || index === 1 || index === 4) {
+                expect(getComputedStyle(body).display).to.equal("block");
+            } else if (index === 2 || index === 3) {
+                expect(getComputedStyle(body).display).to.equal("none");
```

Comment:
You are checking index 2 to have active class just above, and here you are checking the display to be none.Is this correct?